VocaDb.Migrations/Migrations.cs

```diff
@@ -5,15 +5,96 @@ namespace VocaDb.Migrations {
 
 	// Migration version format: YYYY_MM_DD_HHmm
 
+	[Migration(2020_03_01_1300)]
+	public class ArchivedEventVersionsRenameVenue : Migration {
+
+		public override void Up() {
+			Execute.Sql("UPDATE [ArchivedEventVersions] SET ChangedFields = REPLACE(ChangedFields, 'Venue', 'VenueName') WHERE ChangedFields LIKE '%Venue%'");
+		}
+
+		public override void Down() {
+			Execute.Sql("UPDATE [ArchivedEventVersions] SET ChangedFields = REPLACE(ChangedFields, 'VenueName', 'Venue') WHERE ChangedFields LIKE '%VenueName%'");
+		}
+
+	}

```

Comment:
I know I should not use `Execute.Sql`, but I could not find a way other than this.

Reply:
Using Execute.Sql is fine. But I think you should add the "WHERE ChangedFields LIKE '%Venue%' AND NOT ChangedFields LIKE '%VenueName%'... just in case.